var/spack/repos/builtin/packages/boxlib/package.py

```diff
@@ -35,9 +35,9 @@ def cmake_args(self):
                 "-DBL_USE_PARTICLES=1",
                 "-DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON",
                 "-DENABLE_FBASELIB=ON",
-                "-DCMAKE_C_COMPILER=%s" % spec["mpi"].mpicc,
-                "-DCMAKE_CXX_COMPILER=%s" % spec["mpi"].mpicxx,
-                "-DCMAKE_Fortran_COMPILER=%s" % spec["mpi"].mpifc,
+                f"-DCMAKE_C_COMPILER={spec['mpi'].mpicc}",
```

Comment:
also here, just use self.define(key, val)

Reply:
Hadn't used `self.define()` before, but I really like it. We should make sure more folks know about this!